lib/core/address_validator.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter/foundation.dart';
+import 'package:cw_bitcoin/address_validate.dart' as bitcoin;
```

Comment:
we can't use other packages in the main lib (`cw_bitcoin` and `cw_haven` etc...) as it will give a compilation error when building Monero.com as it does Not have bitcoin, haven, nor litecoin.
so please move the `cw_bitcoin/lib/address_validate.dart` to the project directory `lib` as it's not inclusive for BTC wallets only, its for BTC address validation in general, from any wallet
Also renaming it the file would be better to indicate that it validates bitcoin addresses